### PR TITLE
Issue 2 view entry - Daniel Rosenbaum

### PR DESCRIPTION
**Pull request by danielwrosenbaum**

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Daniel! Please provide a screenshot of your output for this feature

**Comment by danielwrosenbaum** [comment]
Screen Shot 2021-02-10 at 4 46 29 PM
Screen Shot 2021-02-10 at 4 46 22 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Daniel! it looks as though your form is still visible when you are on the `entries` view. Please make sure that the view not currently being used is hidden.

**Comment by danielwrosenbaum** [comment]
Screen Shot 2021-02-10 at 4 57 12 PM

File: js/main.js [hunk 1]
```diff
@@ -20,9 +22,47 @@ function theSubmit(event) {
   $submitObj.title = titleVal;
   $submitObj.id = data.nextEntryId;
   data.entries.unshift($submitObj);
+  var newListEntries = journalReturn($submitObj);
+  theList.prepend(newListEntries);
   data.nextEntryId++;
   $photoUrl.src = 'images/placeholder-image-square.jpg';
   $formVar.reset();
+  $formVar.className = 'form hidden';
+  location.assign('#entry');
```
Comment (codingcodymiller) [inline js/main.js]: This is not the best way to do this. You should update the className for the element with `data-view="entries"` to not be hidden, in the same form as you are doing for the `form` on the line above.

File: js/main.js [hunk 2]
```diff
@@ -20,9 +22,47 @@ function theSubmit(event) {
...
+  newTitle.className = 'titles';
+  newTitle.textContent = data.title;
+  var newNotes = document.createElement('p');
+  divColumn2.appendChild(newNotes);
+  newNotes.className = 'entry-notes';
+  newNotes.textContent = data.notes;
+  return newListItem;
+}
+newButton.addEventListener('click', function (event) {
+  location.assign('#newEntry');
```
Comment (codingcodymiller) [inline js/main.js]: Same thing here, you should update the className for the element

File: js/main.js [hunk 3]
```diff
@@ -20,9 +22,47 @@ function theSubmit(event) {
...
+  $formVar.className = 'form';
+});
+window.addEventListener('DOMContentLoaded', function (event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var newListEntries = journalReturn(data.entries[i]);
+    theList.append(newListEntries);
+  }
+});
```
Comment (codingcodymiller) [inline js/main.js:74]: This should be in `data.js`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
🎉 🎉 🎉

**Pull request merged**